client/src/components/ActivityList/ActivityList.css

```diff
@@ -0,0 +1,105 @@
+@import url('https://fonts.googleapis.com/css?family=Reem+Kufi&display=swap');
+
+.ActivityList {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    padding: 0px 40px;
+}
+
+.activitiesContainer {
+    padding: 20px 0px;
+}
+
+.activityHeader {
+    font-family: 'Reem Kufi', sans-serif;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 36px;
+    line-height: 54px;
```

Comment:
Maybe use a heading tag here? That will cut down on the amount of styling you need here.